client/src/partials/references/groups/groups.html

```diff
@@ -1,154 +1,200 @@
-              <td><a class="action danger" ng-click="delete(reference_group)"><span class="glyphicon glyphicon-trash"></span></a></td>
-            </tr>
-            <tr ng-if="!reference_groups.data.length">
-              <td colspan="8">{{ "REFERENCE_GROUP.NO_REFERENCES" | translate }}</td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-    </div>
-
-    <div class="col-xs-5" ng-switch="session.action">
-      <div class="alert alert-info" ng-switch-default>
-        <h4>{{ "REFERENCE_GROUP.TITLE" | translate }}</h4>
-        <p>{{ "REFERENCE_GROUP.ALERT" | translate }}</p>
       </div>
 
-      <div ng-switch-when="new">
-        <form novalidate class="form-horizontal" name="new">
-          <legend><h3>{{ "REFERENCE_GROUP.NEW_REFERENCE" | translate }} </h3></legend>
-
-          <div class="form-group">
-            <label class="control-label col-xs-4 required">{{ "COLUMNS.REFERENCE_GROUP" | translate }}</label>
-            <div class="col-xs-7">
-              <input ng-model="session.new.reference_group" class="form-bhima" required>
-            </div>
-          </div>
-
-          <div class="form-group">
-            <label class="control-label col-xs-4 required">{{ "COLUMNS.LABEL" | translate }}</label>
-            <div class="col-xs-7">
-              <input ng-model="session.new.text" class="form-bhima" required>
-            </div>
-          </div>
-
-          <div class="form-group">
-            <label class="control-label col-xs-4 required">{{ "COLUMNS.POSITION" | translate }}</label>
-            <div class="col-xs-7">
-              <input type="number" ng-model="session.new.position" class="form-bhima" required>
-            </div>
+      <div class="col-xs-6" ng-switch="ReferenceGroupCtrl.view">
+        <div ng-switch-default>
+          <div class="alert alert-info">
+            <h4><STRONG>{{ "REFERENCE_GROUP.TITLE" | translate }}</STRONG></h4>
+            <p>{{ "REFERENCE_GROUP.ALERT" | translate }}</p>
           </div>
+        </div>
 
-          <div class="form-group">
-            <label class="control-label col-xs-4 required">{{ "COLUMNS.BALANCE_SECTION" | translate }}</label>
-            <div class="col-xs-7">
-              <select class="form-bhima" ng-model="session.new.section_bilan_id" required> 
-                <option value="" disabled="disabled">--- {{ "SELECT.BALANCE_SECTION" | translate }} ---</option>
-                <option ng-repeat="section in section_bilan.data | orderBy:'text'" value="{{section.id}}">
-                  {{ section.text }}
-                </option>
-              </select>
-            </div>
+        <!-- Success saving Feed Back -->
+        <div ng-switch-when="create_success">
+          <div class="alert alert-success" id="create_success">
+            <h4>{{ 'REFERENCE_GROUP.SAVE_SUCCESS' | translate }} </h4>
+            </p>
```

Comment:
There are a number of trailing </p> tags throughout this template. This is not a blocking issue however it would be good practice to clean these up.